pkg/cli/admin/release/mirror.go

```diff
@@ -442,18 +451,71 @@ func (o *MirrorOptions) Run() error {
 	} else {
 		fmt.Fprintf(o.Out, "\nSuccess\nUpdate image:  %s\nMirrored to: %s\n", to, o.To)
 	}
+
+	printImageContentInstructions(o.Out, o.From, o.To, repositories)
 	return nil
 }
 
-func sourceImageRef(is *imagev1.ImageStream, name string) (string, bool) {
-	for _, tag := range is.Spec.Tags {
-		if tag.Name != name {
-			continue
-		}
-		if tag.From == nil || tag.From.Kind != "DockerImage" {
-			return "", false
-		}
-		return tag.From.Name, true
+// printImageContentInstructions provides exapmles to the user for using the new repository mirror
+// https://github.com/openshift/installer/blob/master/docs/dev/alternative_release_image_sources.md
+func printImageContentInstructions(out io.Writer, from, to string, repositories map[string]struct{}) error {
+	type installConfigSubsection struct {
```

Comment:
Is it possible this gets called with no entries?

Reply:
If `repositories` had no entries it would mean the release image didn't reference any images.  In this case, only one item would be printed for mirror usage, which is provided by `--from`.

```
imageContentSources:
- mirrors:
  - registry.svc.ci.openshift.org/rteague-1/mirror-test
  source: quay.io/openshift-release-dev/ocp-release
```